bliss/surveys/dc2.py

```diff
@@ -253,6 +145,7 @@ def prepare_data(self):  # noqa: WPS324
                             "bands": self.bands,
                             "n_split": self.n_split,
                             "split_file_path": self.split_file_path,
+                            "min_flux_threshold": self.min_flux_threshold,
```

Comment:
This deeply nested code is somewhat hard to read. Can you define an intermediate variable instead?

Reply:
have fixed it